developers/weaviate/starter-guides/compression.mdx

```diff
@@ -0,0 +1,149 @@
+
+Compressed indexes use less RAM when they are loaded in memory, however they also use more disk space than uncompressed vectors. Weaviate stores the uncompressed vector and the compressed vector index. This means increased disk storage. However, since the cost of RAM is orders of magnitude higher than the cost of disk, the overall cost to use a compressed index is much lower than the cost of using an uncompressed index.
+
+The cost savings are obvious with in memory indexes such as HNSW. The greater the RAM reduction, the smaller the costs.
+
+- Uncompressed data size is roughly 7x greater than PQ compressed data.
+- Uncompressed data size is roughly 4x greater than SQ compressed data.
+- Uncompressed data size is roughly 32x greater than BQ compressed data.
+
+For additional details, refer to the [blog posts](#blog-posts).
+
+#### Recall
+
+Recall measures how well an algorithm finds true positive matches in a data set.
+
+A compressed vector has less information than the full, uncompressed vector. A vector that would match a search might be missed if key information is missing from the compressed vector. That lowers recall.
+
+To improve recall in BQ and SQ, Weaviate over-fetches a list of candidate vectors during a search. Then, for each item on the candidate list, Weaviate searches the corresponding uncompressed vector for a match. This followup search is slower than an in-memory search, but since Weaviate only has to search a limited number of uncompressed vectors, it is still very fast. Most importantly, searching the uncompressed vectors greatly improves recall.
+
+PQ has tunable search parameters that can increase the size of the candidate pool. The mechanism is different, but the effect is the same as over-fetching in SQ and BQ. You can configure your PQ searches to return a vector candidate pool that boosts recall to meet your application requirements.
+
+The over-fetching technique, also call rescoring, means you get the benefits of compression without losing the precision of an uncompressed vector search.
+
+#### Query speed
+
+Compressed vectors are significantly smaller than uncompressed vectors. It is much faster to search a compressed data set than an uncompressed one. After Weaviate creates a candidate list, there is a small time cost to search the uncompressed candidate vectors. This cost is small compared to the cost of searching a large, uncompressed data set, and the improved recall justifies the time spent.
+
+Each compression method has its own characteristics with regard to speed.
+
+- PQ indexes have response rates approach the response rates of uncompressed indexes when recall reaches 97 percent and higher. At those levels of recall, the [speed profile](/blog/pq-rescoring#qps-vs-recall-experiments) for PQ compressed indexes matches the profile for uncompressed indexes. Both indexes handle about 200 queries per second (QPS).
```

Comment:
> Both indexes handle about 200 queries per second (QPS).

The 200 QPS depends on the data and server spec.
We should remove this sentence.

Reply:
We can include benchmarks later. But these will have specific params